packages/gatsby-link/src/index.js

```diff
@@ -73,8 +55,27 @@ class GatsbyLink extends React.Component {
             !e.ctrlKey &&
             !e.shiftKey
           ) {
```

Comment:
Nope, looks great! Just been traveling recently. Merging now and will be prepping new release soon, thanks!